Source/ResourceLib.NetStandard/ResourceLib.NetStandard.csproj

```diff
@@ -0,0 +1,210 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <AssemblyName>Vestris.ResourceLib</AssemblyName>
+    <RootNamespace>Vestris.ResourceLib</RootNamespace>
+    <DefineConstants>NETSTANDARD</DefineConstants>
+    <DocumentationFile>$(OutputPath)\$(AssemblyName).xml</DocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="..\Version\Properties\GlobalAssemblyInfo.cs">
+      <Link>Properties\GlobalAssemblyInfo.cs</Link>
+    </Compile>
+
+    <Compile Include="DeviceIndependentBitmap.cs" />
+    <Compile Include="PixelFormat.cs" />
+
+    <Compile Include="../ResourceLib/Accelerator.cs">
```

Comment:
I would personally put the .csproj next to the normal one to avoid all the `../ResourceLib` in paths. (NUnit does like that https://github.com/nunit/nunit/tree/master/src/NUnitFramework/framework)

Reply:
I can try, but I really prefer keeping each .csproj file in its own directory. It does make things a lot easier, especially with new-style csproj files, Microsoft.NET.Sdk resets the `OutputPath` anyways, which means if I want to have the two in the same directory, I'll have to edit the full-framework csproj. I was trying to touch that code as little as possible.